src/nav/browser.yml

```diff
@@ -1,66 +1,98 @@
+      - title: Cached pages
+        path: /docs/browser/new-relic-browser/page-load-timing-resources/cached-pages
   - title: Installation
     pages:
-      - title: Install browser monitoring
+      - title: Install the browser agent
         path: /docs/browser/browser-monitoring/installation/install-browser-monitoring-agent
       - title: Update the browser agent
         path: /docs/browser/new-relic-browser/installation/update-browser-agent
-      - title: Disable browser monitoring
-        path: /docs/browser/new-relic-browser/installation/disable-browser-monitoring
-      - title: Delete apps from New Relic
-        path: /docs/browser/new-relic-browser/installation/delete-apps-new-relic-browser
+      - title: Compatibility and requirements
+        path: /docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring
   - title: Configuration
     pages:
-      - title: App settings
+      - title: Configure browser settings
         path: /docs/browser/new-relic-browser/configuration/browser-app-settings-page
-      - title: 'Browser Apdex, countries'
+      - title: Set up alerts
+        path: /docs/browser/new-relic-browser/configuration/view-browser-apps-alert-information
+      - title: Upload source maps
+        path: /docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors
+      - title: Filter data by geo location
         path: /docs/browser/new-relic-browser/configuration/browser-settings-ui-options-apdex-geography
-      - title: Privacy settings
-        path: /docs/browser/browser-monitoring/configuration/privacy-settings
-      - title: Rename browser apps
-        path: /docs/browser/new-relic-browser/configuration/rename-browser-apps
-      - title: Domain settings
+      - title: Filter domains and subdomains
         path: /docs/browser/new-relic-browser/configuration/monitor-or-block-specific-domains-subdomains
-      - title: Filter Ajax request events
+      - title: Filter ajaxRequest events
         path: /docs/browser/new-relic-browser/configuration/filter-ajax-request-events
       - title: Group data by URL patterns
         path: /docs/browser/new-relic-browser/configuration/group-browser-metrics-urls
-      - title: License key and app ID
-        path: /docs/browser/browser-monitoring/configuration/browser-license-key-app-id
-      - title: Alerts in browser
-        path: /docs/browser/new-relic-browser/configuration/view-browser-apps-alert-information
-  - title: "Single page app (SPA) monitoring"
+      - title: Disable browser monitoring
+        path: /docs/browser/new-relic-browser/installation/disable-browser-monitoring
+      - title: Rename browser apps
+        path: /docs/browser/new-relic-browser/configuration/rename-browser-apps
+      - title: Delete apps from New Relic
+        path: /docs/browser/new-relic-browser/installation/delete-apps-new-relic-browser
+  - title: Browser monitoring UI
```

Comment:
@paperclypse One idea I have for these docs is to rename the parent title to "Browser monitoring features" or "Monitor browser performance" and transition the child pages from UI overview docs into topic-based articles for each browser data type. For example, "Ajax page" would be "Ajax requests", "JS Errors page" would be "JavaScript errors", etc., and the page would cover how to monitor that aspect of a browser app, the business impact of poor impact, how to interpret Ajax data in the UI, any customizations/troubleshooting they need to know, etc. From an SEO side, it could be impactful when someone searches "monitor javascript errors" and they come across our doc that covers all things JS error monitoring. But this would be larger work. In the meantime, what do you think the parent title be?

Reply:
@a-sassman I do prefer more active titles, like "Monitor browser performance" or even "Monitor your browser page performance". I'd look for opportunities to do that wherever you can.

I think your impulse to reorganize these docs into more task-related categories is a positive one and I wholeheartedly approve!

One thing to keep in mind is that where a doc is in the nav doesn't affect SEO at all. In a search result, the full doc title is returned and not the nav title.